### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by alvra**
This applies fixes to the translation feature. Refer to the individual commits for details.

### CODE REVIEW

**Review by beckerr-rzht — COMMENTED** [review]
Translation is imcomplete now

File: eduvpn/network.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import enum
 from functools import partial
 from time import sleep
+from gettext import gettext
```
Comment (beckerr-rzht) [inline eduvpn/network.py:6]: See above.

File: eduvpn/ui/ui.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import webbrowser
 import logging
 from datetime import datetime
+from gettext import gettext as _, ngettext
```
Comment (beckerr-rzht) [inline eduvpn/ui/ui.py:11]: See above.

**Review by beckerr-rzht — COMMENTED** [review]
Translation is incomplete now.
(Sorry to post this twice. It would be a shame if it would be released like this.)

**Comment by alvra** [comment]
@beckerr-rzht Thanks, the problem is addressed in #349.

**Comment by gijzelaerr** [comment]
@beckerr-rzht can you verify if that PR addresses your issue?

**Comment by beckerr-rzht** [comment]
Yes, it looks perfect! Good job!
looks-good

**Pull request merged**